Add CHVRCHES in Glastonbury 2016

podondra opened the pull request:
Hi. I have just annotated a live concert by CHVRCHES. I hope I didn't mess up. Please make sure that everything is okay.

podondra committed:
Add CHVRCHES in Glastonbury 2016

pnevyk commented:
Hi, looks great! Times of songs I checked seem very accurate. Just two things. First, is there a reason why the name of the band is all capital letters? Second, value of the `File` column should be without the `.txt` extension, it is used as a convenient filename for other formats as well, e.g. downloaded audio file.

Anyway, thanks for your help.

podondra committed:
Remove '.txt' extension from CHVRCHES's metadata

podondra commented:
Thanks for feedback @pnevyk.

Firstly, I think capital letters should be used as they present them self this way on their [music videos](https://www.youtube.com/watch?time_continue=1&v=_mTRvJ9fugM). Still there are some exceptions because on their [website](https://chvrch.es/) they use 'Chvrches'. But I personally would stick to capital letters.

The second one is my fault. I have corrected it also in README and added new commit.

Anything else? :blush:

pnevyk commented:
According to Wikipedia, their name is stylised in capital letters with that special E, so yeah, let's stick with capital letters.

Nice catch with the readme 👍 It was therefore my fault, not yours.

Merging.
